File: src/modelForServerMVC/LoginDao.java
package modelForServerMVC;

public class LoginDao {

	static final String ID = "longlee";
	static final String PASSWD = "6789";
	
	public LoginDao() {
		
	}
	
	public boolean checkLogin(LoginDo loginDo) {
		return ID.equals(loginDo.getId()) && PASSWD.equals(loginDo.getPasswd());
	}
}

File: WebContent/clientMVC/scripts/model/loginDAO.js
var LoginDAO = function() {
	var id = localStorage.getItem('id');
	var passwd = localStorage.getItem('passwd');

	this.getId = function() {
		return id;
	};
	
	this.getPasswd = function() {
		return passwd;
	};
};

LoginDAO.prototype.checkLogin = function(loginDO) {
	var result = false;
	
	if(loginDO.getId() == this.getId() && loginDO.getPasswd() == this.getPasswd()) {
		result = true;
		localStorage.setItem('login', loginDO.getId());
	}
	
	return result;
};

LoginDAO.prototype.logout = function() {
		localStorage.removeItem('login');
		
		return true;
};